Added icons to footer, and SVG folder to main

qod-dante opened the pull request:
I added icons to the footer from [](https://fontawesome.com). The website demanded that I sign in, as well as add a link to their license somewhere on our html page in order to use these icons, otherwise they don't work. All the icons are linked to an external JavaScript that I placed in the <head> section.

I also added a folder for the SVG files (named 'svg') in case we want to use those ones instead of the external ones. Additionally, for those who will handle CSS, I added - via HTML comments - alternative versions of the icons in case they need to be changed.

qod-dante committed:
Added icons to footer, and SVG folder to main

Vusumzi-qod commented on index.html:
```diff
@@ -1,233 +1,242 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href=" https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+  <title>Quick-tik</title>
+  <script src="https://kit.fontawesome.com/91b8d0632f.js" crossorigin="anonymous"></script>
```
Dante please use a link tag instead of a script tag and the link tag should come before the title tag

qod-dante committed:
Updated a minor change

I was unable to find the <link rel="stylesheet"> for fontawesome for some reason. The only change I made this time was to switch the positions line 9:
(<title>Quick-tik</title)
and line 10:
(<script src="https://kit.fontawesome.com/91b8d0632f.js" crossorigin="anonymous"></script>)
At Gilfoyle's insistence.
